src/Random/README.md

```diff
@@ -0,0 +1,62 @@
+# Dafny.Random
+
+## Overview
+
+The `Dafny.Random` module provides a uniform interface to random values across target languages.
+
+```
+    // Return an arbitrary boolean value
+    predicate nextBool()
+
+    // Return an arbitrary integer value in the range [0, bound)
+    function nextInt(bound: int := 0): (value: int)
+        ensures 0 <= value
+        ensures bound > 0 ==> value < bound
+
+    // Return an arbitrary real value in the range [0,1)
+     function nextReal(): (value: real)
```

Comment:
This does seem useful, but I'd call it something more specific, since the implicit bounds will be less obvious on the caller side.